contracts/src/v0.8/ccip/rmn/RMNRemote.sol

```diff
@@ -256,11 +251,11 @@ contract RMNRemote is Ownable2StepMsgSender, ITypeAndVersion, IRMNRemote, IRMN {
   /// @inheritdoc IRMNRemote
   function isCursed() external view override(IRMN, IRMNRemote) returns (bool) {
     // There are zero curses under normal circumstances, which means it's cheaper to check for the absence of curses.
-    // than to check the subject list twice, as we have to check for both the legacy and global curse subjects.
+    // than to check the subject list twice, as we have to check for the global curse subjects.
```

Comment:
comment not correct anymore

also nit: there was a period after the first line which should not have been there